fire_perimeter/active_fire.py

```diff
@@ -31,16 +32,15 @@ def apply_cloud_cover_threshold(start_date, n_days, cloud_threshold):
 def apply_classification_rule(data):
     # get DEM, LandCover, Sentinel-2 "L2A" (level two atmospherically-
     # corrected "bottom of atmosphere (BOA) reflectance) data """
-    nasa_dem = ee.Image('NASA/NASADEM_HGT/001').select('elevation') 
+    nasa_dem = ee.Image('NASA/NASADEM_HGT/001').select('elevation')
     land_cover = ee.ImageCollection("ESA/WorldCover/v100").first()
-    
 
     # apply classification rule
-    rule = 'R > G && R > B && (LC != 80) && (LC != 50) && (LC != 70) && (DEM < 1500)'
+    rule = 'x = R > G && R > B && (LC != 80) && (LC != 50) && (LC != 70) && (DEM < 1500)'
```

Comment:
What does the "x=" do?

Reply:
It names the resultant band. I was trying to convince earth engine to give me a 20m resolution image, by asking for one band (e.g. less total file size).